From ffc6388b3b8d94e84bfec81e41bb609057619f76 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 12 Jan 2021 17:57:55 +0200
Subject: [PATCH] [SW-0] ppboot: several white space fixes

White space changes - no functional changes.
Several moves of last parameters of funciton calls to the same line the
function resides for better readability.
Those lines are less than 80 characters, so we can have all the
parameters on the function call line.
Remove double white spaces.

Change-Id: I8185ada68481fc53cc4023d22ad95e683a956c0f
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/ppboot/ppboot.c b/ppboot/ppboot.c
index 4c63b71..2297620 100644
--- a/ppboot/ppboot.c
+++ b/ppboot/ppboot.c
@@ -33,8 +33,7 @@
 	u32 idx, bit;
 	char *msg;
 
-	idx = boot_desc_get_img_desc(image_pending,
-				     HL_FW_IMG_PREBOOT, desc);
+	idx = boot_desc_get_img_desc(image_pending, HL_FW_IMG_PREBOOT, desc);
 	if (idx == HL_FW_IMG_NOT_FOUND)
 		return false;
 
@@ -44,15 +43,14 @@
 	else
 		preboot_puts_raw("candidate");
 
-	preboot_puts_hex_raw(" (offset: ", desc->spi_offset,
-			false);
+	preboot_puts_hex_raw(" (offset: ", desc->spi_offset, false);
 	preboot_puts_hex_raw(", size: ", desc->img_size, false);
 	preboot_puts(")\n");
 
 	ppboot_cp_preboot(desc->loadaddr, desc->spi_offset,
-			desc->img_size, "PREBOOT");
+			  desc->img_size, "PREBOOT");
 	if (desc->desc_ver > 1) {
-		is_valid  = ppboot_is_valid_img_desc(desc);
+		is_valid = ppboot_is_valid_img_desc(desc);
 		if (!is_valid)
 			preboot_puts("Checksum is not valid.\n");
 	}
